### ISSUE TITLE
examples/lm_finetuning/simple_lm_finetuning.py crashes with cublas runtime error

### ISSUE THREAD

**Issue by fredriko** [huggingface/pytorch-transformers#1255]
### TL;DR
I'm trying to finetune the existing English **bert-base-uncased** model according to the examples in `examples/lm_finetuning/README.md` on IMDB data, but fail. The `simple_lm_finetuning.py` script crashes with a cublas runtime error.

### VERBOSE

I have tried the following on a local machine, as well as on GCP, with two different datasets.

I have formatted the input data according to the specification in `examples/lm_finetuning/README.md` and stored it in a file called `imdb_corpus_1.txt` (essentially using the information kindly provided in https://medium.com/dsnet/running-pytorch-transformers-on-custom-datasets-717fd9e10fe2 wrt to the data and preprocessing)

To reproduce the issue, run the following on a suitable dataset.

The command:
```
~/pytorch-transformers/examples/lm_finetuning$ python3 simple_lm_finetuning.py --train_corpus ~/imdb_corpus_1_small.txt --bert_model bert-base-uncased --do_lo
wer_case --output_dir finetuned_lm --do_train
```

results in output ending with the following stacktrace:
```
raceback (most recent call last):
  File "simple_lm_finetuning.py", line 641, in <module>
    main()
  File "simple_lm_finetuning.py", line 591, in main
    outputs = model(input_ids, segment_ids, input_mask, lm_label_ids, is_next)
  File "/home/fredrik_olsson/venv/pytorch-transformers/lib/python3.7/site-packages/torch/nn/modules/module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/fredrik_olsson/venv/pytorch-transformers/lib/python3.7/site-packages/pytorch_transformers/modeling_bert.py", line 694, in forward
    head_mask=head_mask)
  File "/home/fredrik_olsson/venv/pytorch-transformers/lib/python3.7/site-packages/torch/nn/modules/module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/fredrik_olsson/venv/pytorch-transformers/lib/python3.7/site-packages/pytorch_transformers/modeling_bert.py", line 623, in forward
    head_mask=head_mask)
  File "/home/fredrik_olsson/venv/pytorch-transformers/lib/python3.7/site-packages/torch/nn/modules/module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/fredrik_olsson/venv/pytorch-transformers/lib/python3.7/site-packages/pytorch_transformers/modeling_bert.py", line 344, in forward
    layer_outputs = layer_module(hidden_states, attention_mask, head_mask[i])
  File "/home/fredrik_olsson/venv/pytorch-transformers/lib/python3.7/site-packages/torch/nn/modules/module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/fredrik_olsson/venv/pytorch-transformers/lib/python3.7/site-packages/pytorch_transformers/modeling_bert.py", line 322, in forward
    attention_outputs = self.attention(hidden_states, attention_mask, head_mask)
  File "/home/fredrik_olsson/venv/pytorch-transformers/lib/python3.7/site-packages/torch/nn/modules/module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/fredrik_olsson/venv/pytorch-transformers/lib/python3.7/site-packages/pytorch_transformers/modeling_bert.py", line 279, in forward
    self_outputs = self.self(input_tensor, attention_mask, head_mask)
  File "/home/fredrik_olsson/venv/pytorch-transformers/lib/python3.7/site-packages/torch/nn/modules/module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/fredrik_olsson/venv/pytorch-transformers/lib/python3.7/site-packages/pytorch_transformers/modeling_bert.py", line 199, in forward
    mixed_query_layer = self.query(hidden_states)
  File "/home/fredrik_olsson/venv/pytorch-transformers/lib/python3.7/site-packages/torch/nn/modules/module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/fredrik_olsson/venv/pytorch-transformers/lib/python3.7/site-packages/torch/nn/modules/linear.py", line 87, in forward
    return F.linear(input, self.weight, self.bias)
  File "/home/fredrik_olsson/venv/pytorch-transformers/lib/python3.7/site-packages/torch/nn/functional.py", line 1371, in linear
    output = input.matmul(weight.t())
RuntimeError: cublas runtime error : resource allocation failed at /pytorch/aten/src/THC/THCGeneral.cpp:216
```

*However*, the following command, also pertaining to finetuning, successfully executes:

```
~/pytorch-transformers/examples$ python3 run_lm_finetuning.py --train_data_file ~/imdb_corpus.txt --output_dir fredriko --model_name_or_path bert-base
-uncased --mlm --do_train --do_lower_case --evaluate_during_training --overwrite_output_dir
```

## Environment

* OS: ubuntu
* Python version: python 3.7.3
* PyTorch version: torch 1.2.0
* PyTorch Transformers version (or branch): pytorch-transformers 1.2.0
* Using GPU ? yes
* Distributed of parallel setup ? no
* Any other relevant information:

**Comment by Zacharias030** [comment]
Maybe related to the change in order of parameters to BertModel's forward method ?
See #1246

**Comment by thomwolf** [comment]
Hi, the lm finetuning examples are now replaced by `examples/run_lm_finetuning`
